public/js/index.js

```diff
@@ -1,5 +1,31 @@
-function displayMessage() {
-  return 'Hello, World!';
+document.body.appendChild(buildProducts(products));
+
+function buildProducts(products) {
+  var div = document.createElement("div");
```

Comment:
Styling guide on the Wiki says : 

> Use single quotes ('string') for strings.

I've already changed it in the pull request though.